generator/uscore/templates/unit_tests/unit_test.rb.erb

```diff
@@ -12,8 +12,8 @@ describe Inferno::Sequence::<%= class_name %> do
     @token = 'ABC'
     @instance = Inferno::Models::TestingInstance.create(url: @base_url, token: @token, selected_module: '<%= module_name %>')
     @client = FHIR::Client.for_testing_instance(@instance)
-    @patient_id = 'example'
-    @instance.patient_id = @patient_id
+    @patient_ids = 'example'
```

Comment:
Is it possible to replace the other locations where `'example'` is hard coded with this instance variable?

Reply:
Yep good point. All other hardcoded 'example' locations are changed to '@sequence.patient_ids.first'.